app/views/content/train-to-be-a-teacher/classroom-placements.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+navigation: 20.20
+navigation_title: School placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-itt
```

Comment:
Would it be better to have an adviser promo here? People may well be more committed if reading this guidance?